client/src/modules/employees/registration/employees.html

```diff
@@ -0,0 +1,513 @@
+                      <i class="fa fa-warning"></i>
+                      {{ "FORM.LABELS.NO_GRADE" | translate }} <b>{{ "FORM.LABELS.CONTACT_ADMIN" | translate }}</b>
+                    </uib-alert>
+
+                    <label class="col-md-3 control-label">{{ "FORM.LABELS.GRADE" | translate }}</label>
+                    <div class="col-md-9">
+                      <ui-select
+                        name="grade"
+                        ng-model="EmployeeCtrl.employee.grade_id"
+                        required>
+                        <ui-select-match placeholder="{{ 'FORM.SELECT.GRADE' | translate }}">
+                          <span>{{$select.selected.format}}</span>
+                        </ui-select-match>
+                        <ui-select-choices ui-select-focus-patch repeat="g.uuid as g in EmployeeCtrl.grades | filter:{format : $select.search} | orderBy:'text'">
+                          <span ng-bind-html="g.format | highlight:$select.search"></span>
+                        </ui-select-choices>
+                      </ui-select>
+
+                      <div class="help-block" ng-messages="DetailForm.grade.$error" ng-show="EmployeeRegistrationForm.$submitted">
+                        <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+                      </div>
+                    </div>
+                  </div>
+
+                  <!-- Employee Hospital File Number -->
+                  <div
+                    class="form-group has-feedback"
+                    ng-class="{'has-error' : DetailForm.hospitalNumber.$invalid && EmployeeRegistrationForm.$submitted || DetailForm.hospitalNumber.$error.unique}">
+
+                    <label class="col-md-3 control-label">{{ "FORM.LABELS.HOSPITAL_FILE_NR" | translate }} </label>
+                    <div class="col-md-9">
+                      <input
+                        name="hospitalNumber"
+                        class="form-control"
+                        ng-model="EmployeeCtrl.employee.hospital_no"
+                        bh-unique="/patients/hospital_number"
+                        ng-maxlength="EmployeeCtrl.length20"
+                        autocomplete="off"
+                        required>
+                      <span ng-show="DetailForm.hospitalNumber.$error.unique" class="glyphicon glyphicon-remove form-control-feedback" id="unique-error-icon"></span>
+                      <span ng-show="DetailForm.hospitalNumber.$pending.unique" class="glyphicon glyphicon-hourglass form-control-feedback"></span>
+
+                      <div class="help-block" ng-messages="DetailForm.hospitalNumber.$error" ng-show="DetailForm.$submitted || DetailForm.hospitalNumber.$error.unique">
+                        <p ng-messages-include="modules/templates/messages.tmpl.html"></p>
+                      </div>
+                    </div>
+                  </div>
+
+                  <!-- Employee creditor group-->
```

Comment:
From the hospital perspective, a employee is a creditor so we need to know what will be his group of creditor because every creditor must have a group.